app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1> Mechanic: <%= @mechanic.name %></h1>
+<p> Years of Experience: <%= @mechanic.years_experience %> </p>
+
+<h3> Current rides they're working on: </h3>
+<% @mechanic.rides.each do |ride| %>
+    <h3><%= ride.name %></h3>
+<% end %>
+
+
+<h3> Add a ride to workload: </h3>
+<%= form_with url: mechanic_path(@mechanic), method: :post do |f| %>
```

Comment:
You might want to make thte label and text_field called `ride_id` just to be very explicit and not confuse the 2 IDs!